Configuration/TypoScript/Library/themes.basic.constantsts

```diff
@@ -78,3 +78,8 @@ themes.configuration.contacts.phone = 555-555555
 
 # cat=theme,basic/contacts; type=string; label= E-Mail address
 themes.configuration.contacts.email = info@info.se
+
+# customsubcategory=maps= Google map configuration
+
+# cat=theme,basic/maps; type=string; label= Google map api key
+themes.configuration.maps.map = //maps.googleapis.com/maps/api/js?key=AIzaSyARuWux_ERbn3FIzy6Wm36WwY8IU8Ih9yg
```

Comment:
@MattiasNilsson @dmh,

It is not the default public key, I think it need to be change in every project of t3kit. I have read some topic related to Google map api It is limit 2,500 free requests per day, so we cannot use key that i already generated.

More detail:
https://developers.google.com/maps/faq?csw=1#usagelimits
https://developers.google.com/maps/documentation/geocoding/usage-limits